client/pages/ResetPassword/index.js

```diff
@@ -0,0 +1,122 @@
+import React, { useContext, useState } from 'react';
+import { Formik } from 'formik';
+import axios from 'axios';
+import InputForm from '@components/InputForm';
+import Button from '@components/Button';
+import { ResetPasswordValidator } from '@clientValidators/Auth';
+
+const ResetPassword = () => {
+    const [errorState, setErrorState] = useState(false);
```

Comment:
Please get rid of all `state` values just like i mentioned above.

Reply:
Done